### PR TITLE
initial commit of spectools README

### PR DESCRIPTION
**Pull request by dericed**
Readme based on https://mailarchive.ietf.org/arch/msg/cellar/7g4t_MetEmSxmJToGDbG_aXt52s

### CODE REVIEW

File: spectool/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- tabler
+
+# Installation
+
+Building spectools is dependant on coremake. To build for make run these commands from ~/foundation-source.
+
+```
+cd corec/tools/coremake
+make
+sudo make install
```
Comment (robUx4) [inline spectool/README.md]: Normally you don't need to install coremake. It can run from the folder where the sources are.

Comment (mbunkus) [inline spectool/README.md]: No, you cannot. There's a hard-coded path lookup for `/usr/local/share/coremake` when reading the `.build` files. Therefore you have to install it once or use symlink trickery.

Comment (mbunkus) [inline spectool/README.md]: See [lines 5541 ff](https://github.com/Matroska-Org/foundation-source/blob/master/corec/tools/coremake/coremake.c#L5541) and [lines 5556 ff](https://github.com/Matroska-Org/foundation-source/blob/master/corec/tools/coremake/coremake.c#L5556) for what I mean. You don't have to install it on Windows but on all other platforms.

Comment (robUx4) [inline spectool/README.md]: If anything I can autotools it.

**Pull request merged**